Wazo 3350 - Remove queueskill category (#424)
wazo-platform/wazo-confd: CHANGELOG.md

  * POST `/1.1/contexts/<context_id>/ranges/<range type>`. It can be used to list available extensions in a given context.

> * the `agent skill` resource now has no more `category` field.

Reviewer: ```suggestion
* the `agent skill` resource no longer has a `category` field.
```
